src/templates/homepage/AnnouncementsSection.tsx

```diff
@@ -112,7 +113,9 @@ export const TemplateAnnouncementsSection = forwardRef<
                                   ]
                                 }
                               >
-                                {announcement.date}
+                                {moment(announcement.date, "DD/MM/YYYY").format(
+                                  "DD MMMM YYYY"
+                                )}
```

Comment:
noted that this will reconcile the diff in view but it still feels weird to have it here (as a view level thing) rather than just doing it at the data level. this offloads responsibility to the dev next time to know this @__@

Reply:
352dda83e9ad59e741de145591ada8630fdc7fa1